### PR TITLE
update subscription tree serializer

### PR DESCRIPTION
**Pull request by utkarshparkhi**
Fixed subscription tree serializer by checking the ancestor of the current category for subscribed parameter @pradumangoyal

### CODE REVIEW

**Comment by pradumangoyal** [comment]
@utkarshparkhi be reminded of our conventions for commit messages:
- It should be in the present tense
- The first letter should be capital
- Should tell a little about what changes were made, besides the file name

File: utils/getSubscription.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+from categories.models import Category,UserSubscription
+
+class getSubscription:
+    should_subscribe = {}
```
Comment (Ajayneethikannan) [inline utils/getSubscription.py]: If they aren't used, can you remove these class variables

File: utils/getSubscription.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+from categories.models import Category,UserSubscription
+
+class getSubscription:
```
Comment (Ajayneethikannan) [inline utils/getSubscription.py]: Capital G here 😅

**Review by Ajayneethikannan — APPROVED** [review]

File: utils/getSubscription.py (hunk 2 above)
Comment (pradumangoyal) [reply utils/getSubscription.py]: Also have you made the corresponding changes in notifications and emails? Like removing this file and renaming the class according to this.

Comment (pradumangoyal) [reply utils/getSubscription.py]: Filename: `get_subscripion.py`?

File: utils/getSubscription.py (hunk 1 above)
Comment (utkarshparkhi) [reply utils/getSubscription.py]: they are used i thought it was better to declare them as class variable rather than assing them as function parameters

**Pull request merged**